packages/domains-table/src/utils/paths.ts

```diff
@@ -0,0 +1,124 @@
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
Looks like I didn't something similar to this `src/utils/paths.ts` file here https://github.com/Automattic/wp-calypso/pull/81120 😳

I think it'll be easier to let this PR finish.